routes/modelExport.js

```diff
@@ -1,9 +1,33 @@
-var linz = require('../');
+'use strict';
+
+const linz = require('../');
+const setTemplateScripts = require('../lib/scripts');
```

Comment:
So `linz.api.views.getScripts(req, res)` would get the list of scripts for that route?

Linz updates the `res.locals` after the user has access to it, by exposing it at the last second we can allow the user to edit whatever they want on the page. In v1 we would update all the Marko templates to use `out.global.somelocalproperty` which contains req, res, app, app.locals, and res.locals http://markojs.com/docs/express/#usage